src/main/scala/djinni/Main.scala

```diff
@@ -179,7 +179,7 @@ object Main {
       opt[String]("objc-include-prefix").valueName("<prefix>").foreach(objcIncludePrefix = _)
         .text("The prefix for #import of header files from Objective-C files.")
       opt[String]("objc-swift-bridging-header").valueName("<name>").foreach(x => objcSwiftBridgingHeaderName = Some(x))
-        .text("The name of Objective-C Bridging Header used in XCode's Swift projects.")
+        .text("The name of Objective-C Bridging Header used in XCode's Swift projects. The output folder is --objc-header-out.")
```

Comment:
Documentation for the output directory of the bridging header in `djinni --help`